x-pack/plugins/uptime/public/components/functional/monitor_list.tsx

```diff
@@ -79,16 +80,20 @@ export const MonitorList = ({ dangerColor, loading, monitors, primaryColor }: Mo
           },
           {
             field: 'ping.monitor.id',
-            name: i18n.translate('xpack.uptime.monitorList.idColumnLabel', {
-              defaultMessage: 'ID',
+            name: i18n.translate('xpack.uptime.monitorList.nameIdColumnLabel', {
```

Comment:
Thank you for updating key. Can you please add a description as well?
The key for it is `description`, it's a sibling of `defaultMessage` and it takes a string.